openapi/aca-openapi-v2.json

```diff
@@ -0,0 +1,892 @@
+{
+  "openapi" : "3.0.1",
+  "info" : {
+    "title" : "ACA-Upload-API-v2",
+    "description" : "Microservice to manage PagoPA GPD Upload",
+    "termsOfService" : "https://www.pagopa.gov.it/",
+    "version" : "0.1.22"
```

Comment:
check the right version https://pagopa.github.io/pagopa-status-page/, maybe it could be only a generation bug